src/Uno.UWP/Storage/Helpers/StorageFileHelper.iOSmacOS.cs

```diff
@@ -0,0 +1,28 @@
+﻿#nullable enable
+#if __IOS__ || __MACCATALYST__ || __MACOS__
+using System.Reflection;
+using System.Threading;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Foundation;
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Helpers;
+
+	partial class StorageFileHelper
```

Comment:
Seems this is indented even though the namespace is file-scoped - can you please unindent the rest of the file?